src/data/validation/email.js

```diff
@@ -111,10 +111,10 @@ is invalid. Please try again.`;
   return errorsDict;
 };
 
-const validateEmailTemplateForm = (formData, templateKey) => {
+const validateEmailTemplateForm = (formData, templateKey, subjectRequired = true) => {
```

Comment:
nit: Consider keeping naming consistent with the above `validateEmailTemplateFields()` method by using `isSubjectRequired`?

Reply:
Fair point